modules/cms/classes/Theme.php

```diff
@@ -484,6 +494,7 @@ public static function resetCache()
 
         Cache::forget(self::ACTIVE_KEY);
         Cache::forget(self::EDIT_KEY);
+        Cache::forget(self::CONFIG_KEY);
```

Comment:
You'd need to add a call to resetCache in writeConfig() to match the other logic